### PR TITLE
SAML documentation: add a section on steps to join a federation

### LINKED ISSUE #21: Declare what the SAML AssertionConsumerServiceURL is

### PR DESCRIPTION
**Pull request by jouvin**
Promise a while ago... adding it to upcoming 1.8.0 documentation?

Hopefully fixes #21

### CODE REVIEW

File: content/en/docs/reference/configuration/external-authentication/saml/_index.md [hunk 1]
```diff
@@ -187,6 +187,33 @@ To limit authentication only to IdPs that are [SIRFTI][sirtfi]-compliant, set
...
+* Wait for the federation metadata to be updated at the federation IdPs. This
+typically involves a 12h delay.
+* Check if login in IAM works successfully through the federation: note that it
+will not work before the federation IdPs have updated their metadata. If the
+error persists after the delay, contact your federation or IdP support.
+
+When registering the Indigo IAM SP, there are a few critical parameters to supply:
+
+* Assertion Consumer Service (ACS), consisting of 3 parameters:
+  * Binding: muste be `HTTP Post`
```
Comment (rmiccoli) [inline content/en/docs/reference/configuration/external-authentication/saml/_index.md]: Fix typo with **must**

File: content/en/docs/reference/configuration/external-authentication/saml/_index.md [hunk 2]
```diff
@@ -187,6 +187,33 @@ To limit authentication only to IdPs that are [SIRFTI][sirtfi]-compliant, set
...
+  * URL: it is the instance URL + `/saml/SSO`, e.g. `https://iam.example.com/saml/SSO`
+  * Index: should be set to 0
+* X509 SAM certificate: the generated `self-signed` certificate
```
Comment (rmiccoli) [inline content/en/docs/reference/configuration/external-authentication/saml/_index.md]: Fix typo with x509 **SAML** certificate

Comment (jouvin) [reply content/en/docs/reference/configuration/external-authentication/saml/_index.md]: Fixed

File: content/en/docs/reference/configuration/external-authentication/saml/_index.md (hunk 1 above)
Comment (jouvin) [reply content/en/docs/reference/configuration/external-authentication/saml/_index.md]: Fixed

**Pull request merged**